src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java

```diff
@@ -6,91 +6,97 @@
 /**
  * Some Javadoc.
  * <p>
- * /^ WARN/   Some Javadoc.<p>
- *
+ * /^ WARN/ Some Javadoc.<p>
  */
-// 2 violations 3 lines above:
+// 2 violations 2 lines above:
 //  '\<p\> tag should be placed immediately before the first word'
 //  '\<p\> tag should be preceded with an empty line.'
 
 class InputIncorrectJavadocParagraph {
 
+  // 2 violations 4 lines below:
+  //  '\<p\> tag should be placed immediately before the first word'
+  //  '\<p\> tag should be preceded with an empty line.'
+  /**
+   * Some Javadoc.<p>
```

Comment:
I'm curious: was the `P` changed to lowercase by google formatter?

Reply:
Yes. Formatter changed it to lowercase and also fixed its position but we had to show the violation so I reverted back its position but did not converted it back to the uppercase